src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -230,7 +230,7 @@ public function addXmlContent($content, $charset = 'UTF-8')
         $dom->validateOnParse = true;
 
         if ('' !== trim($content)) {
-            @$dom->loadXML($content, LIBXML_NONET);
+            @$dom->loadXML($content, LIBXML_NONET | LIBXML_PARSEHUGE);
```

Comment:
This constant defined in php extension and avail since PHP >= 5.3.2 and PHP >= 5.2.12 which is less than min requirement for Symfony 2